keyboards/axolstudio/yeti/config.h

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 kb-elmo
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x05E4
```

Comment:
This needs to be changed, as the VID is in use. 
https://www.the-sz.com/products/usbid/index.php?v=0x05E4&p=&n=

```
❯ echo -n "Axolstudio" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
525C
```

```suggestion
#define VENDOR_ID    0x525C
```